core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemApiTest.java

```diff
@@ -54,10 +54,10 @@ public void after() {
 
   @Test
   public void unknownAuthorityTriggersWarning() throws IOException {
-    URI unknown = URI.create("alluxio://test/");
+    URI unknown = URI.create("alluxio://test:/");
```

Comment:
@qian0817 Is this because `getUriDefaultPort()` method is deleted above? Could you please explain why you needed this change? Ideally add this description here in the code as comment.

Reply:
This was added because previously the regex for logical authorities used `([^:,;]+)`. Now it is `([a-zA-Z_\\-0-9.]+)` so I think you could use something like `alluxio://te$t/`.

Reply:
After using the `ebj@` prefix, I revert it.